### ISSUE TITLE
Improve performance of workloads list view

### ISSUE THREAD

**Issue by nwmac** [rancher/dashboard#6457]
Performance can be impacted when there is frequent pod activity

**Comment by nwmac** [comment]
Fixed by two PRs:

https://github.com/rancher/dashboard/pull/6390
https://github.com/rancher/dashboard/pull/6388

See https://github.com/rancher/dashboard/pull/6390 for instructions on how to test

**Comment by git-ival** [comment]
@nwmac @bmdepesa 
I was able to test this on the latest 2.6-head (ee084d0) and performance looks good. However I encountered a scenario that caused the Restart count and Health columns to refresh but not finish loading even after waiting for a few minutes.

In the case that the user is filtering the deployments and updates a given deployment the restart count and health columns begin to load but do not seem to finish loading after waiting for a few minutes even though the Deployment had already finished updating. Here is a short video that shows this behavior.
Outside of this, LGTM.

https://user-images.githubusercontent.com/85186645/184458162-37ee1097-149f-457a-99df-321e8b6f02b3.mp4
